src/main/resources/theme/style-webview.css

```diff
@@ -16,7 +16,7 @@ html, body {
 
 a {
   /* Use same as -fx-accent */
-  color: #009688;
+  color: #3F51B5;
```

Comment:
We should make this a variable

Reply:
I agree, but this is a different css file. I don't know if that is possible.